### PR TITLE
Update link.php

### PR DESCRIPTION
**Pull request by kapetanov**
the pagnation in has an misuse of the aria-label. i fixed it with two span tags.

Pull Request for Issue # .

### CODE REVIEW

**Comment by brianteeman** [comment]
Sorry I dont understand this change

**Comment by kapetanov** [comment]
the aria-label does not work in span tags. that's why i put the value of the aria-label in a hidden span tag. assistive technology will be able to read this span tag. the original span tag is shown on normal displays and its hidden (aria-hidden="true") for assistive technology. thats why i use two span tags. one for assistive technology and one for displays.

**Comment by brianteeman** [comment]
> the aria-label does not work in span tags.

it does if its an interactive element - I agree that its not interactive here.

> that's why i put the value of the aria-label in a hidden span tag

this is what I dont understand - it is still a non interactive span so it will still be ignored

**Comment by brianteeman** [comment]
### Before
image

### After
image

image

**Comment by kapetanov** [comment]
you need the value of the aria label because otherwise only the number of the page are readable for assistive technologies. that would be too little for people with visual impairment. for example, instead of '1' for assistive technologies, 'page 1' should be readable.

**Comment by brianteeman** [comment]
> only the number of the page are readable for assistive technologies. that would be too little for people with visual impairment.

I agree

The problem is that I do not see how your change will do that. The screenshots show that its not exposed in the accessibility tree

**Comment by kapetanov** [comment]
ok, i see what u mean

**Comment by kapetanov** [comment]
i think its the hidden css of bootstrap

**Comment by kapetanov** [comment]
ok, the current page must also be an link (a tag). not a span tag. then the aria-label will work (active element). tomorrow i made a new workaround.

**Comment by brianteeman** [comment]
Not completely tested but this should work

`		<a aria-current="true" aria-label="<?php echo $aria; ?>" href="#" class="page-link"><?php echo $display; ?></a>
`

**Comment by richard67** [comment]
@kapetanov It seems this PR is made to solve your issue #35758 . if that is right, you should have mentioned that issue at the top of your PR description, i.e. „Pull request for issue #35758 “. Please add that, and please close your issue with reference to the pull request here. Thanks in advance.

**Comment by kapetanov** [comment]
@brianteeman yes, it works. i have tested it.

File: layouts/joomla/pagination/link.php [hunk 1]
```diff
@@ -91,7 +91,7 @@
 <?php elseif (isset($item->active) && $item->active) : ?>
 	<?php $aria = Text::sprintf('JLIB_HTML_PAGE_CURRENT', strtolower($item->text)); ?>
 	<li class="<?php echo $class; ?> page-item">
-		<span aria-current="true" aria-label="<?php echo $aria; ?>" class="page-link"><?php echo $display; ?></span>
+		<span aria-current="true" class="hidden"><?php echo $aria; ?></span><span aria-hidden="true" class="page-link"><?php echo $display; ?></span>
```
Comment (brianteeman) [inline layouts/joomla/pagination/link.php]:
```suggestion
		<a aria-current="true" aria-label="<?php echo $aria; ?>" href="#" class="page-link"><?php echo $display; ?></a>
```

**Comment by patvb2003** [comment]
I have tested this item :white_check_mark: successfully on 19a3ddd7085750d21b89d0143210b7f9a535b0ab<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35771">issues.joomla.org/tracker/joomla-cms/35771</a>.</sub>

**Comment by kiki-G** [comment]
I have tested this item :white_check_mark: successfully on 19a3ddd7085750d21b89d0143210b7f9a535b0ab<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35771">issues.joomla.org/tracker/joomla-cms/35771</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35771">issues.joomla.org/tracker/joomla-cms/35771</a>.</sub>

**Comment by chmst** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35771">issues.joomla.org/tracker/joomla-cms/35771</a>.</sub>

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**